migration.json

```diff
@@ -963,6 +1020,26 @@
         "OrganizationRegister": "0xbF9D0eb0419CDfc72CC674aa0f64D85FB2AEc7E8",
         "Redeemer": "0xc1597A0Ca76AA007B3f3Ad783a6A60562F709e25",
         "UGenericScheme": "0x72dB54E63B3004a4bd030E5E0333579C1d54B863"
+      },
+      "0.0.1-rc.31": {
+        "GEN": "0x543Ff227F64Aa17eA132Bf9886cAb5DB55DCAddf",
+        "DAORegistry": "0xE9f949fee2ae09a408C9b902031bC2A1BFF3AEe2",
+        "DAOTracker": "0xD4b8A4a032AD063A4FF3f39E563267179a51B1e4",
+        "ControllerCreator": "0x0bC4fca7a3F9f8A397D953c38d5E7E9506D99905",
+        "DaoCreator": "0xA796ddb9aC5C22629c0f97447dA9eb23B4F3C6cC",
+        "UController": "0xeAa8Db4E80F6DD5041b080EDd735A24E4bEAa1EC",
+        "GenesisProtocol": "0x35984c26D661d69fB589844202358B3e6c438a3b",
+        "SchemeRegistrar": "0xCe77Cb61fbF2600F0e29Bb50Be3a5B32Fc61a0a4",
+        "UpgradeScheme": "0x91E9E40CB74020aE2a16a3f0cf57ac2df00aB4D4",
+        "GlobalConstraintRegistrar": "0x07099013F7E407B676b57CA37DDFA02557130ca2",
+        "ContributionReward": "0xB723dd7ca4f4E533Eb9966AfB8C083a5a7655095",
+        "AbsoluteVote": "0xf5dFa5160565B99Dd0972Aab98C25Dd02E05635a",
+        "QuorumVote": "0xc9ebacdFaA355273331bB1DF37C3ac20987240c2",
+        "TokenCapGC": "0x349900Ed029559448d9E28D65c3e79C27685920f",
+        "VoteInOrganizationScheme": "0xa70E04B6237e88FE2F3111C981820AF228Aa7F7a",
+        "OrganizationRegister": "0x0d46D76b3d02Ea3BEdBea68C4208d61E4fB2B303",
+        "Redeemer": "0x7102a3e2b4aD1163b6f52B5aA6463f158E5c5146",
+        "UGenericScheme": "0x67883470ae73347a6742376D3db8BF13bc9358a8"
```

Comment:
why this has new address ?

Reply:
because the code (more correctly imported contracts code) changed.